### PR TITLE
test: updated flaky mobilview test

### PR DESCRIPTION
**Pull request by NandanAnantharamu**

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/MobileResponsiveTests/HugWidgetValidationSpec.js [hunk 1]
```diff
@@ -1,37 +1,28 @@
...
-    cy.wait(2000);
     cy.get(commonlocators.refreshApp).click({
       force: true,
     });
     cy.wait(2000);
-    cy.addDsl(dsl);
+    cy.get("canvas").should("be.visible");
+    cy.addDsl(dsl); //for dsl to settle
+    cy.get(".t--widget-imagewidget").first().should("be.visible");
     cy.wait(5000); //for dsl to settle
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/MobileResponsiveTests/HugWidgetValidationSpec.js:19]: Wait & then check if imagewidget is loaded. If image widget is visible by this line of execution, then following wait is not needed!

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/MobileResponsiveTests/FillwidgetResizingSpec.js [hunk 2]
```diff
@@ -1,35 +1,27 @@
...
+    cy.get(commonlocators.convert).should("be.visible").click({
       force: true,
     });
-    cy.wait(2000);
-    cy.get(commonlocators.refreshApp).click({
+    cy.get(commonlocators.refreshApp).should("be.visible").click({
       force: true,
     });
-    cy.wait(2000);
+        cy.wait(2000);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/MobileResponsiveTests/FillwidgetResizingSpec.js:17]: There are many GitHub Actions/build failures. Please correct.

**Comment by Aishwarya-U-R** [comment]
@NandanAnantharamu - Add appropriate PR description to this.

**Comment by NandanAnantharamu** [comment]
closing this as there is another PR now

**Pull request closed without merging**